keyboards/delikeeb/waaffle/keymaps/default/keymap.c

```diff
@@ -0,0 +1,134 @@
+/* Copyright 2020 noclew
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _LOWER,
+    _RAISE,
+    _ADJUST
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    BASE,
+    LOWER,
+    RAISE,
+    RGBF,
+    RGBR,
+    VOLUP,
+    VOLDN
```

Comment:
```suggestion
```

Reply:
I am using these four custom keycodes in the keymaps. Isn't it that the keymap will not work if I remove these lines?

Reply:
I'm not sure why these need to be custom keycodes, they seem to just be defined as already existing RGB and volume keycodes...